cmd/composer-cli/blueprints/save_test.go

```diff
@@ -63,6 +73,10 @@ func TestCmdBlueprintsSave(t *testing.T) {
 
 	cmd, out, err := root.ExecuteTest("blueprints", "save", "simple")
 	defer out.Close()
+	defer func() {
```

Comment:
This is not needed. The blueprint is saved into a temporary directory which is removed on exit from the function. See lines 54-72

Reply:
I am absolutely positive that I had a "simple.toml" laying around while writing the other 2 tests and I've added this to get rid of it. Maybe it has to do with the weir behavior where the values aren't switched between tests and I was seeing failures related to that. IDK.  Reverted.